components/list/list-item-drag-mixin.js

```diff
@@ -1,20 +1,76 @@
+import { css, html } from 'lit-element/lit-element.js';
 import { announce } from '../../helpers/announce.js';
+import { dragActions } from './list-item-drag-handle.js';
 import { getUniqueId } from '../../helpers/uniqueId.js';
-import { html } from 'lit-element/lit-element.js';
 import { nothing } from 'lit-html';
 
+const move = Object.freeze({
+	above: true,
+	below: false
+});
+
+const timeDelayForLeavingTheDropArea = 1000; //ms
+
 export const ListItemDragMixin = superclass => class extends superclass {
 
 	static get properties() {
 		return {
 			draggable: { type: Boolean, reflect: true },
-			key: { type: String, reflect: true }
+			dragText: { type: String, attribute: 'drag-text' },
+			key: { type: String, reflect: true },
+			dragging: { type: Boolean, reflect: true, attribute: 'dragging' },
+			_bottomPlacementMarker: { type: Boolean },
+			_dropTarget: { type: Boolean },
+			_keyboardMode: { type: Boolean },
+			_topPlacementMarker: { type: Boolean }
```

Comment:
I think this `_topPlacementMarker` and `_bottomPlacementMarker` are named like objects/elements but are actually booleans, which makes the code a little harder to understand.  These booleans are derived from the state object's `insertBefore` boolean, and is therein set by `setDestination(destination, insertBefore)`.  Ultimately the value comes from the `move` enum (which being a verb, is named like a method).  

I think some of this can be cleaned up by:

* renaming `move` to `moveLocations` or perhaps even `dropLocation` might be more appropriate
* updating `setDestination(destination, insertBefore)` signature to be `setDestination(destination, dropLocation)`
* updating `_insertBefore` in the state object to be `_dropLocation`
* replacing `_topPlacementMarker`and `_bottomPlacementMarker` with `_dropLocation`

Basically, I think if we name this stuff consistently then it makes it easier to follow.